src/main/java/components/HeaderComponent.java

```diff
@@ -0,0 +1,4 @@
+package components;
+
+public class HeaderComponent {
```

Comment:
we decided to put elements of this component in the abstractPageObject, aren't we?

Reply:
Done.
Thank You!